src/locales/de/splash-messages.json

```
@@ -25,12 +25,120 @@
   "moreAddictiveThanIntended": "Süchtig machender als beabsichtigt!",
   "mostlyConsistentSeeds": "Meistens konsistente Seeds!",
   "achievementPointsDontDoAnything": "Errungenschaftspunkte tun nichts!",
-  "youDoNotStartAtLevel": "Du startest nicht auf Level 2000!",
-  "dontTalkAboutTheManaphyEggIncident": "Wir reden nicht über den Manaphy-Ei-Vorfall!",
+  "nothingBeatsAJellyFilledDonut": "Nothing Beats a Jelly-Filled Donut!",
+  "dontTalkAboutTheTinkatonIncident": "Don't Talk About the Tinkaton Incident!",
```

```suggestion
  "nothingBeatsAJellyFilledDonut": "Donuts mit Marmelade, es gibt nicht besseres auf der Welt!",
  "dontTalkAboutTheTinkatonIncident": "Wir reden nicht über den Granforgita-Vorfallt!",
```

The Donut one is hard. The german version of this definitly isnt that memeable. 
But Rocko (Brock) says "es gibt nicht besseres auf der Welt" to them so idk